editor/src/components/canvas/canvas-strategies/strategies/keyboard-reorder-strategy.ts

```diff
@@ -108,10 +109,11 @@ function fitness(interactionSession: InteractionSession | null): number {
   }
 
   const accumulatedPresses = accumulatePresses(interactionSession.interactionData.keyStates)
-  const matches = accumulatedPresses.some((accumulatedPress) =>
-    Array.from(accumulatedPress.keysPressed).some(
-      (key) => key === 'left' || key === 'right' || key === 'up' || key === 'down',
-    ),
+  const matches = accumulatedPresses.some(
+    (accumulatedPress) =>
+      Array.from(accumulatedPress.keysPressed).some(
+        (key) => key === 'left' || key === 'right' || key === 'up' || key === 'down',
+      ) && shallowEqual(accumulatedPress.modifiers, emptyModifiers),
```

Comment:
We have an equality function for modifiers here: <https://github.com/concrete-utopia/utopia/blob/master/editor/src/utils/modifiers.ts#L89-L100>

Reply:
thanks! fixed in [2c7b067](https://github.com/concrete-utopia/utopia/pull/3389/commits/2c7b067e2b06fe17a5d2d77624360f0e59a1e241)